src/mixins/instanceMixins/JOIous.js

```diff
@@ -30,7 +30,7 @@ const JOIous = ReceivingClass => class extends ReceivingClass {
   }
 
   static deepEquals(objA, objB) {
-    const objDiff = this.diff(objA, objB)
+    const objDiff = this.diff(objA.serialize(), objB.serialize())
```

Comment:
only the serialized results count as equality. Anything not communicated in serialization is internal to the fctgraph workings, and not persisted when it is sent/stored as JSON